app/views/post/index.html.erb

```diff
@@ -1,3 +1,7 @@
-<h1>Post#index</h1>
-<p>Find me in app/views/post/index.html.erb</p>
-<p>Watch Euphoria</p>
\ No newline at end of file
+<%= render partial: "partials/user", locals: { user: @user } %>
+<% @posts.each do |post| %>
+  <%= render partial: "partials/post", locals:{ post: post } %>
+  
+  <%# post comments %>
+  <%= render partial: "partials/comment", locals: { items: 5 } %>
+<% end %>
```

Comment:
- [ ] Uses `post.recent_comments` to render 5 most recent comments in the `app/views/posts/index.html.erb` page